docs/getting_started_guide/Installation_Options/ISO/ISO_Build/Building_SIMP_From_Tarball.rst

```diff
@@ -128,6 +128,6 @@ Once the process completes, you should have a bootable SIMP ISO, in:
 ``build/distributions/<OS>/<Release>/<Arch>/SIMP_ISO/``
 
 .. _SIMP artifacts repository: https://download.simp-project.com/simp/ISO
-.. _latest 6.2.0-0 release (for EL 6): https://download.simp-project.com/simp/ISO/tar_bundles/SIMP-6.2.0-0.el6-CentOS-6-x86_64.tar.gz
-.. _latest 6.2.0-0 release (for EL 7): https://download.simp-project.com/simp/ISO/tar_bundles/SIMP-6.2.0-0.el7-CentOS-6-x86_64.tar.gz
+.. _latest 6.4 release (for EL 6): https://download.simp-project.com/simp/ISO/tar_bundles/SIMP-6.4.0-1.el6-CentOS-6-x86_64.tar.gz
```

Comment:
This tar does not exist.  One with '-0.el6' does exist.  Should this text be fixed or is the file missing?

Reply:
Good catch; _neither_ of them should have been 6.4.0-1.